Assets/Scripts/DataController.cs

```diff
@@ -138,7 +145,77 @@ void Start()
                 }
             }
         }
+
+        if (markerPrefab != null)
+        {
+            AddMarker("NCP", 0f, 90f, localSiderialTime, colorOrange);
+            AddMarker("SCP", 0f, -90f, localSiderialTime, colorOrange);
+            AddMarker("VE", 0f, 0f, localSiderialTime, colorGreen);
+            AddCircumferenceMarker("equator", colorBlue, markerLineWidth);
+            AddLineMarker("poleLine", colorOrange, GameObject.Find("NCP"), GameObject.Find("SCP"), markerLineWidth);
+        }
     }
+
+    void AddMarker(string name, float RA, float dec, double lst, Color color)
```

Comment:
Should probably rename the "name" property to markerName

Reply:
Good idea.  That will make it clearer when we're requesting the name.

Reply:
Although note in this case I am trying to set the Unity `name` property of the game object.

Reply:
Anyway, changed to markerName.